### ISSUE TITLE
add API point for Daily sumary

### ISSUE THREAD

**Issue by privatemaker** [tracking-exposed/facebook.tracking.exposed#11]
while working on issue i am in need of API point that shows the **daily topics and details** as discussed in issue #7 maybe this already exist and i just don't know?

**Comment by vecna** [comment]
I guess the metadata you need are the one returned in the enrich API https://github.com/tracking-exposed/facebook/issues/128#issuecomment-487613320 (but in this description, it is mean for public data. for this I've to implement the `userToken `check),

This is a working API: https://facebook.tracking.exposed/api/v2/en/enrich/30-0
It return 30 posts, good enough to proceed, and in the meanwhile I'll address this issue. This can help in verify if the format is good for your need, or if more metadata should be added.

**Comment by vecna** [comment]
I've commited a fixture file which simulate to be 2019-06-20. It contains dates of 28 August and so on, don't be surprised. The name is fixed and as long as the fixture stay the same, the link would works. It respond at: http://localhost:1313/fixtures/personal/testing-token/daily/2019-06-20.json

**Comment by vecna** [comment]
I read your message about the confusion, I'll clarify.

**requirement**: an API which authenticate a fbtrex user, and return semantics + metadata

how to do it?

I should do the API, and yes, is on my TODO. but luckily, I found an API which the exact metadata we need. is the _enrich API_. so I told you which is the API I used, and created a fixture to help development.

I aim to don't change the format, but you in the meanwhile can tell me if there is anything confusing in the JSON, or something you might want computed server side. 

The current fixture, 2018-06-20.json, is not personal, but it might be sufficient enough to let you continue the development, and to me to:

1) implement the API 
2) get feedback from you about the data format

hope this clarify!
